app/views/notifier/new_user.html.haml

```diff
@@ -2,8 +2,19 @@
 
 - if @password
   %p
+    - if text(".username_and_password_text").present?
+      = text(".username_and_password_text")
+      %br
```

Comment:
would line 5 still be necessary if we just move this `br` tag into the locale string?